app/src/components/widgets/charts/cumulativeTrendChart/cumulativeTrendChart.jsx

```diff
@@ -63,6 +66,10 @@ export class CumulativeTrendChart extends PureComponent {
     }
   }
 
+  componentWillUnmount() {
+    this.props.observer.unsubscribe('widgetResized', this.toggleControlsVisibility);
```

Comment:
`Cumulative trend chart` can't be resized.
See the comments from https://jiraeu.epam.com/browse/EPMRPP-39637